### ISSUE TITLE
package github.com/heidi-ann/ios: no buildable Go source file

### ISSUE THREAD

**Issue by heidi-ann** [heidi-ann/ios#3]
Resolve ``package github.com/heidi-ann/ios: no buildable Go source file`` to avoid confusion

**Comment by dgryski** [comment]
`go get` works, there's just nothing to build.  Installation instructions can have
`github.com/heidi-ann/ios/...` instead (`...` means "and everything under here.")

Related,

```
work/src/cvs/gocode/src/github.com/heidi-ann/ios/simulator/simulator.go:14: too few values in struct initializer
work/src/cvs/gocode/src/github.com/heidi-ann/ios/simulator/simulator.go:37: too few values in struct initializer
```

**Comment by dgryski** [comment]
`go get` still downloads the source.  Change the instructions to `go get github.com/heidi-ann/ios/...` instead.

`...` means "and everything under this directory".

Related, just filed https://github.com/heidi-ann/ios/pull/4 to fix the build failure for the simulator.

**Comment by dgryski** [comment]
Also, tests seem pretty broken (including build failures):

```
<dgryski@kamek[ios] \ʕ◔ϖ◔ʔ/ > go test ./...
?   	github.com/heidi-ann/ios/api/interactive	[no test files]
?   	github.com/heidi-ann/ios/api/rest	[no test files]
?   	github.com/heidi-ann/ios/cache	[no test files]
?   	github.com/heidi-ann/ios/client	[no test files]
?   	github.com/heidi-ann/ios/config	[no test files]
# github.com/heidi-ann/ios/consensus
consensus/consensus_test.go:17: too few values in struct initializer
consensus/consensus_test.go:31: unknown field 'Request' in struct literal of type msgs.Entry
consensus/consensus_test.go:48: invalid operation: reply != prepare1_res (mismatched types msgs.Prepare and msgs.PrepareResponse)
consensus/consensus_test.go:73: invalid operation: reply != commit1_res (mismatched types msgs.Commit and msgs.CommitResponse)
FAIL	github.com/heidi-ann/ios/consensus [build failed]
# github.com/heidi-ann/ios/msgs
msgs/msgs_test.go:24: unknown field 'Request' in struct literal of type Entry
msgs/msgs_test.go:54: invalid operation: reply != prepare (struct containing Entry cannot be compared)
FAIL	github.com/heidi-ann/ios/msgs [build failed]
?   	github.com/heidi-ann/ios/server	[no test files]
# github.com/heidi-ann/ios/test
test/generator_test.go:52: too few values in struct initializer
test/generator_test.go:59: assignment count mismatch: 2 = 3
--- FAIL: TestSimulator (0.02s)
	simulator_test.go:21: Participant not responding
	simulator_test.go:21: Participant not responding
	simulator_test.go:21: Participant not responding
	simulator_test.go:21: Participant not responding
	simulator_test.go:21: Participant not responding
	simulator_test.go:21: Participant not responding
	simulator_test.go:21: Participant not responding
	simulator_test.go:21: Participant not responding
	simulator_test.go:21: Participant not responding
	simulator_test.go:21: Participant not responding
FAIL
FAIL	github.com/heidi-ann/ios/simulator	0.018s
ok  	github.com/heidi-ann/ios/store	0.003s
FAIL	github.com/heidi-ann/ios/test [build failed]
```

**Comment by heidi-ann** [comment]
Thanks @dgryski.  
I had not updated the simulation/test code base when added functionality to ios. This is now fixed in 4b6a8005f5822c8c90e93bd73ea049d86db200c8.
